src/app/config/config.c

```diff
@@ -3180,6 +3166,20 @@ options_validate_cb(const void *old_options_, void *options_, char **msg)
                   &world_writable_control_socket) < 0)
     return -1;
 
+#ifndef HAVE_SYS_UN_H
```

Comment:
We should be able to easily test these options in a unit test.

We could also test them in test_parseconf.sh, with a git pattern that accepts either log message.